pkg/processor/trigger/http/test/suite/http_test.go

```diff
@@ -192,6 +192,19 @@ func (suite *HTTPTestSuite) getHTTPDeployOptions() *platform.CreateFunctionOptio
 	return createFunctionOptions
 }
 
+func (suite *HTTPTestSuite) TestFooHeader() {
+	fooValue := "fooValue"
+	suite.TestSuite.PlatformConfiguration.Foo = &fooValue
+	createFunctionOptions := suite.getHTTPDeployOptions()
+
+	suite.DeployFunctionAndRequest(createFunctionOptions, &Request{
+		ExpectedResponseHeadersValues: map[string][]string{
+			"X-Nuclio-Foo": {fooValue},
+		},
+	})
+
+}
```

Comment:
That's nice!
Also add a test case where you change the foo value in the function configuration, and see that it is populated properly (not only the default value)